.github/workflows/deploy.yml

```diff
@@ -18,10 +18,15 @@ jobs:
           python --version
           python -m pip install --upgrade pip
           pip install setuptools wheel twine
+      - name: Test
```

Comment:
I figured we should run this before build and publish step. Im do not believe the pip install will affect the follow step wheere we publish.

Reply:
Let's check after I release vivarium later today to see if this affects the published version of vivarium